usage/rest-api/src/main/java/brooklyn/rest/api/ServerApi.java

```diff
@@ -87,6 +88,11 @@ public void shutdown(
             multiValueResponse = false)
     public String getStatus();
 
+    @GET
+    @Path("/getConfig/{configKey}")
+    @ApiOperation(value = "Get the value of the specified config key from brooklyn properties")
+    public String getConfig(@PathParam("configKey") final String configKey);
```

Comment:
Suggest using something like that below (taken from `EntityConfigApi`):

    public String getConfig(
            @ApiParam(value = "Config key ID", required = true)
            @PathParam("configKey") String configKey)